samples/dlang/source/app.d

```diff
@@ -0,0 +1,68 @@
+import std.stdio;
```

Comment:
if this is a sample, it should maybe be `samples/sample_binary.d` to conform to the other languages